style.css

```diff
@@ -527,227 +527,318 @@ form > input {
     display: flex;
-    justify-content: center;
-    height: 100px;
-    border-bottom: 1px solid #dddfe6;
+    text-align: center;
   }
-}
 
-.desktop-menu {
-  display: flex;
-}
+  #hello {
+    display: grid;
+    grid-template-columns: 50% 50%;
+  }
 
-#hello {
-  display: grid;
-  grid-template-columns: 50% 50%;
-}
+  .menu {
+    display: none;
+    visibility: hidden;
+  }
 
-.info-area {
-  background-image: url(./images/image_dots.svg);
-  background-position: bottom right;
-  background-repeat: no-repeat;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  padding: 0 90px 0 160px;
-}
+  .icon {
+    display: none;
+    visibility: hidden;
+  }
 
-.menu {
-  display: none;
-  visibility: hidden;
-}
+  .geo-image {
+    height: 877px;
+    width: 60%;
+    left: 738px;
+    top: 100px;
+    border-radius: 0;
+  }
 
-.icon {
-  display: none;
-  visibility: hidden;
-}
+  .desktopmenu ul {
+    display: flex;
+    list-style: none;
+  }
 
-.geo-image {
-  display: flex;
-  grid-row: 1/5;
-  grid-column: 2/3;
-  margin: 0;
-  padding: 0;
-  height: 100%;
-  width: 100%;
-  border-radius: 0px;
-  background-position: right-top;
-}
+  a {
+    text-decoration: none;
+    color: black;
+    text-align: center;
+    margin-left: 110px;
+  }
 
-main {
-  display: block !important;
-}
+  .desktopmenu li {
+    margin: 20px 50px;
+    align-items: center;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    text-align: right;
+    color: #1a2236;
+    word-spacing: 1px;
+  }
 
-.desktopmenu ul {
-  display: flex;
-  list-style: none;
-}
+  #dot-image {
+    width: 100%;
+    height: 175px;
+    left: 563px;
+    margin-top: 40px;
+    margin-left: 240px;
+  }
 
-.desktopmenu li {
-  margin: 20px 50px;
-  font-size: 16px;
-  align-items: center;
-}
+  .info-area {
+    padding-left: 18px;
+  }
 
-.form-section {
-  align-items: center;
-}
+  .my-name {
+    height: 220px;
+    width: 484px;
+    left: 160px;
+    border-radius: nullpx;
+    font-family: "Playfair Display", sans-serif;
+    font-size: 110px;
+    font-weight: 400;
+    line-height: 110px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: 150px;
+  }
 
-input {
-  align-items: center;
-}
+  .detail {
+    height: 84px;
+    width: 484px;
+    left: 160px;
+    border-radius: nullpx;
+    font-family: "Poppins", sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 21px;
+    letter-spacing: -0.01em;
+    text-align: left;
+    margin-top: 60px;
+  }
 
-.socialicons {
-  margin-right: 50px;
-}
+  .work {
+    display: none;
+    visibility: hidden;
+  }
 
-.my-name {
-  font-size: 6rem;
-  line-height: 100%;
-  margin-top: 15vh;
-}
+  .head-button {
+    width: 29%;
+    height: 70px;
+    left: 160px;
+    background-color: #1a2236;
+    border-radius: 0;
+    padding-left: 20px;
+    margin-top: 30px;
+    margin-right: 20px;
+  }
```

Comment:
- [optional] Please kindly give this button element tect color of #fff as we have it in figma. 👍